README.md

```diff
@@ -118,6 +118,11 @@ _[recover care](https://care.rcvr.app/)_ is a version of recover for institution
 BUILD_VARIANT=care npm run dev
 ```
 
+### Updating supported browsers
+
+After you updated the `.browserslistrc` you need to run `npm run supportedBrowsers` to update the
+regular expression the app is using to figure out if a browser is supported.
```

Comment:
Do you think we could have this generated bit `next.config.js` like

```js
webpack(config, { isServer }) {

    if (isServer) {
      updateSupportedBrowsers()
    }

    return config
},
```
or maybe just add `npm run supportedBrowsers` to the `build` script in `package.json`.

Reply:
Yes, you are right. Let me check where the best place would be.